pinconfigs/gd32f470xx.yml

```diff
@@ -0,0 +1,1229 @@
+# GD32F470XX pin definitions
+#
+# Sources:
+# - GD32F470XX Datasheet (Revision 1.1)
+#
+# Pin codes:
+#
+# - 176 pins: I
+# - 144 pins: Z
+# - 100 pins: V
+#
+# Memory codes:
+#
+# - 512Kb  Flash, 256Kb SRAM: E
+# - 1024Kb Flash, 256Kb SRAM: G
+# - 2048Kb Flash, 512Kb SRAM: I
+# - 3072Kb Flash, 256Kb SRAM: K
+#
+# Copyright (c) 2022, Rtone.
```

Comment:
this is mostly a copy of the f450xx file, so copyrights should be kept.

Reply:
> this is mostly a copy of the f450xx file, so copyrights should be kept.

Sure, no problem.